src/analyzer/vocab.py

```diff
@@ -34,7 +32,7 @@ def corpus_statistics(self):
             word_freq[token] = self.dictionary.cfs[id]
 
         stats["most_common"] = word_freq.most_common(10)
-        stats["rare_words"] = rare_tokens
+        stats["rare_words"] = len(rare_tokens)
```

Comment:
is the idea that we're more interested in the volume of rare words than the specific words? Can you remind me how "rare words" are defined here?

Reply:
Rare words here is defined as words that occur only once, in the Baleen sample I was working with there were a lot of them so for now I changed it to a count to make the output more manageable, although it might be interesting to still capture them.